packages/ncids-js/src/components/nci-header/utils/mobile-menu/mobile-menu.ts

```diff
@@ -411,24 +473,26 @@ export class MobileMenu {
 	}
 
 	/**
-	 * Create custom events for NCIMobileMenu.
+	 * Create custom events for the mobile menu.
 	 *
-	 * The default settings for NCISiteAlert exposes these events:
+	 * The default settings for the mobile menu exposes these events:
 	 * - nci-header:mobile-menu:open
 	 * - nci-header:mobile-menu:close
+	 * - nci-header:mobile-menu:linkclick
 	 *
 	 * @private
 	 */
 	private createCustomEvents(): void {
 		const events = ['close', 'open'];
 		[...events].forEach((event) => {
-			this.customEvents[event] = new CustomEvent(
-				`nci-header:mega-menu:${event}`,
-				{
-					bubbles: true,
-					detail: this.element,
-				}
-			);
+			this.customEvents[event] = (detail: unknown) =>
```

Comment:
can linkclick be added and consolidated into this forEach?

Reply:
yes